fip.c

```diff
@@ -330,9 +366,11 @@ struct fip_toc_info {
  *
  * @param fip: fill up with FIP informations
  * @param fd: FIP image to read header from
+ * @param rev: FIP Format revision
+ * @param bl2sz: BL2 Size
  * @return: 0 on success, negative number otherwise
  */
-static int fip_read_toc(struct fip_toc_info *toc, int fd)
+static int fip_read_toc(struct fip_toc_info *toc, int fd, enum fip_rev rev, size_t bl2sz)
```

Comment:
Can you please do that in two lines in order to follow 80 characters line limit ?